examples/tree_3d_dgsem/elixir_mhdmultiion_ec.jl

```diff
@@ -0,0 +1,61 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal MHD equations
+equations = IdealGlmMhdMultiIonEquations3D(gammas = (1.4, 1.667),
+                                           charge_to_mass = (1.0, 2.0))
+
+initial_condition = initial_condition_weak_blast_wave
+
+volume_flux = (flux_ruedaramirez_etal, flux_nonconservative_ruedaramirez_etal)
+surface_flux = (flux_ruedaramirez_etal, flux_nonconservative_ruedaramirez_etal)
+solver = DGSEM(polydeg = 3, surface_flux = surface_flux,
+               volume_integral = VolumeIntegralFluxDifferencing(volume_flux))
+
+coordinates_min = (-2.0, -2.0, -2.0)
+coordinates_max = (2.0, 2.0, 2.0)
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level = 4,
+                n_cells_max = 10_000)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver,
+                                    source_terms = source_terms_lorentz)
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+analysis_interval = 10
+analysis_callback = AnalysisCallback(semi, interval = analysis_interval)
+alive_callback = AliveCallback(analysis_interval = analysis_interval)
+
+save_solution = SaveSolutionCallback(dt = 0.1,
+                                     save_initial_solution = true,
+                                     save_final_solution = true,
+                                     solution_variables = cons2prim)
+
+cfl = 0.5
+
+stepsize_callback = StepsizeCallback(cfl = cfl)
+
+glm_speed_callback = GlmSpeedCallback(glm_scale = 0.5, cfl = cfl)
+
+callbacks = CallbackSet(summary_callback,
+                        analysis_callback, alive_callback,
+                        save_solution,
+                        stepsize_callback,
+                        glm_speed_callback)
+
+###############################################################################
+# run the simulation
+
+sol = solve(ode, CarpenterKennedy2N54(williamson_condition = false),
+            dt = 1.0, # solve needs some value here but it will be overwritten by the stepsize_callback
+            save_everystep = false, callback = callbacks);
```

Comment:
```suggestion
            ode_default_options()..., callback = callbacks);
```
We recently switched to use `ode_default_options()` instead of `save_everystep = false`.

Reply:
For some reason, the tests pass with `save_everystep = false`, but not with `ode_default_options()...`. I get the following error message:
```julia
  Got exception outside of a @test
  LoadError: MethodError: Cannot `convert` an object of type Bool to an object of type Vector{Vector{Float64}}
  The function `convert` exists, but no method is defined for this combination of argument types.

  Closest candidates are:
    convert(::Type{Array{T, N}}, ::StaticArraysCore.SizedArray{S, T, N, N, Array{T, N}}) where {S, T, N}
     @ StaticArrays ~/.julia/packages/StaticArrays/2dZx4/src/SizedArray.jl:88
    convert(::Type{Array{T, N}}, ::StaticArraysCore.SizedArray{S, T, N, M, TData} where {M, TData<:AbstractArray{T, M}}) where {T, S, N}
     @ StaticArrays ~/.julia/packages/StaticArrays/2dZx4/src/SizedArray.jl:82
    convert(::Type{T}, ::T) where T
     @ Base Base.jl:126
    ...

```

Reply:
You probably need a semicolon `;` after `CarpenterKennedy2N54(williamson_condition = false)`

Reply:
That fixed it. Thanks!